sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -829,7 +835,7 @@ def begin_analyze_batch_actions(  # type: ignore
         except ValueError as error:
             if "API version v3.0 does not have operation 'begin_analyze'" in str(error):
                 raise ValueError(
-                    "'begin_analyze_batch_actions' endpoint is only available for API version v3.1-preview.3"
+                    "'begin_analyze_batch_actions' endpoint is only available for API version v3.1-preview.4"
```

Comment:
can you add a test for pii categories too? You can also add it to samples if you think it's necessary

Reply:
I added a couple of tests.  One thing I'm still unsure about is why using `domain=phi` seems to recognize all entities, even the ones that, according to the official [docs](https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/named-entity-types?tabs=personal#azure-information), are not usually supposed to be returned as PHI.  Since the interaction between domain and pii_categories is an intersection, I'd eventually like to write a test to show that there is at least one combination that will cause a list of zero entities to be returned, but I need to know which entities are not supposed to be classified as PHI.

For samples, we may want to consider adding a new one to show this interaction between domain and categories, but seems like it doesn't fit well with the existing sample.  Let me know if this is something we should do for this release or if we should wait to get more info from the service team about the issue I mentioned above about PHI.